tests/Doctrine/Tests/DBAL/Functional/Ticket/DBAL3691Test.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Functional\Ticket;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Platforms\MariaDb1027Platform;
+use Doctrine\DBAL\Schema\MySqlSchemaManager;
+use PHPUnit\Framework\TestCase;
+use ReflectionMethod;
+
+/**
+ * @group DBAL-3691
+ */
+class DBAL3691Test extends TestCase
+{
+    public function testIssue()
+    {
+        $conn     = $this->createMock(Connection::class);
+        $platform = new MariaDb1027Platform();
+
+        $schemaManager = new MySqlSchemaManager($conn, $platform);
+
+        $reflectionMethod = new ReflectionMethod($schemaManager, '_getPortableTableColumnDefinition');
+        $reflectionMethod->setAccessible(true);
+
+        $column = $reflectionMethod->invoke($schemaManager, [
```

Comment:
This type of a test doesn't really tell why this change is needed. It just proves that internally the code works as expected. Could you please provide a functional test that would cover all supported platforms and potentially find the same issue in others?

Reply:
Sorry, but I guess this issue is outdated and was already fixed. Is it OK to close this PR?

Reply:
Sure, feel free to close it.